tests/regtest/mod.rs

```diff
@@ -0,0 +1,6 @@
+#![cfg(feature = "regtest")]
+mod bitcoin;
+mod chain_swaps;
+mod liquid;
+
+const WAIT_TIME_MS: i32 = 5_000;
```

Comment:
Just fyi: I guess we can remove or reduce that value after after https://github.com/BoltzExchange/boltz-backend/pull/848 is merged